people_data.py

```diff
@@ -0,0 +1,71 @@
+import re
+from datetime import datetime
+
+class PeopleData:
+    _id_counter = 1
```

Comment:
We won't be required to generate ID in code in the future as primary keys are set automatically. But let it be for now

Reply:
Ok, got it